Renovator having issue moving a folder with AEM 6.3.3.3

## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 6.3.3.3
* [x] ACS AEM Commons Version: 4.1.0
* [x] Reproducible on Latest? yes


### Expected Behavior

_Please describe briefly the expected behavior, i.e. when I do X, Y should happen._
Using the WKND project, I try to move the folder /content/wknd/en to /content/wknd/en-new.

### Actual Behavior

_Please describe briefly the actual behavior, i.e. when I do X, Z happens._
Instead of the folder moving, I get an error message in Renovator.
### Steps to Reproduce

_Starting from a vanilla instance of AEM at the above version, describe in explicit steps how to reproduce. Link to
any supporting assets, e.g. content packages, below_
1) Starting with AEM 6.3, I installed SP3 and CFP 3.  AEM version should be 6.3.3.3.
2) Install ACS common 4.1.0.
3) Install the WKND project from Adobe.
4) Start up Renovator and configure Source to be /content/wknd/en and Destination to be /content/wknd/en-new.
5)  Publish: None, no other option select, press Start.
6)  I get the follow error message

2019-06-07 11:27:26	Moving /content/wknd/en	Error when moving node /content/wknd/en: Not supported yet

Stack trace
com.adobe.acs.commons.mcp.impl.processes.renovator.MovingException: Error when moving node /content/wknd/en: Not supported yet. at com.adobe.acs.commons.mcp.impl.processes.renovator.MovingPage.move(MovingPage.java:84) at com.adobe.acs.commons.mcp.impl.processes.renovator.Renovator.lambda$null$25(Renovator.java:533) at com.adobe.acs.commons.mcp.impl.processes.renovator.Renovator$$Lambda$207/1248142964.accept(Unknown Source) at com.adobe.acs.commons.fam.impl.ActionManagerImpl.withResolver(ActionManagerImpl.java:200) at com.adobe.acs.commons.fam.impl.ActionManagerImpl.runActionAndLogErrors(ActionManagerImpl.java:167) at com.adobe.acs.commons.fam.impl.ActionManagerImpl.lambda$deferredWithResolver$0(ActionManagerImpl.java:159) at com.adobe.acs.commons.fam.impl.ActionManagerImpl$$Lambda$149/1052258358.run(Unknown Source) at com.adobe.acs.commons.fam.i...(TimedRunnable.java:91) at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) at java.util.concurrent.Future...(FutureTask.java:266) at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) at java.lang.Thread.run(Thread.java:745) Caused by: java.lang.UnsupportedOperationException: Not supported yet. at com.adobe.acs.commons.mcp.impl.processes.renovator.ReplicatorQueue.getActivatedPaths(ReplicatorQueue.java:86) at com.day.cq.wcm.core.impl.PageManagerImpl.countActivatedPaths(PageManagerImpl.java:2215) at com.day.cq.wcm.core.impl.PageManagerImpl.isSelfOrDescendantActivated(PageManagerImpl.java:2132) at com.day.cq.wcm.core.impl.PageManagerImpl.move(PageManagerImpl.java:449) at com.adobe.acs.commons.mcp.impl.processes.renovator.MovingPage.moveOrClonePage(MovingPage.java:91) at com.adobe.acs.commons.mcp.impl.processes.renovator.MovingPage.lambda$move$0(MovingPage.java:80) at com.adobe.acs.commons.mcp.impl.processes.renovator.MovingPage$$Lambda$208/668524527.accept(Unknown Source) at com.adobe.acs.commons.fam.actions.Actions.lambda$retry$2(Actions.java:141) at com.adobe.acs.commons.fam.actions.Actions$$Lambda$209/1246045493.accept(Unknown Source) at com.adobe.acs.commons.mcp.impl.processes.renovator.MovingPage.move(MovingPage.java:82) ... 12 more

### Links

_Links to related assets, e.g. content packages containing test components_

--- a/bundle/src/main/java/com/adobe/acs/commons/json/JcrJsonAdapter.java
+++ b/bundle/src/main/java/com/adobe/acs/commons/json/JcrJsonAdapter.java
@@ -95,7 +95,7 @@ public class JcrJsonAdapter extends TypeAdapter<Node> {
 
     @Override
     public Node read(JsonReader reader) throws IOException {
-        throw new UnsupportedOperationException("Not supported yet.");
+        throw new UnsupportedOperationException("JcrJsonAdaper.read(JsonReader) is not supported yet.");
     }
 
 }

--- a/bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/renovator/ReplicatorQueue.java
+++ b/bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/renovator/ReplicatorQueue.java
@@ -83,7 +83,7 @@ public class ReplicatorQueue implements Replicator {
 
     @Override
     public Iterator<String> getActivatedPaths(Session session, String s) throws ReplicationException {
-        throw new UnsupportedOperationException("Not supported yet.");
+        return getActivateOperations().keySet().iterator();
     }
 
     /**
@@ -92,8 +92,8 @@ public class ReplicatorQueue implements Replicator {
     @Override
     @Deprecated
     public List<ReplicationContentFilter> createContentFilterChain(ReplicationAction action) {
-        throw new UnsupportedOperationException("Not supported yet.");
-    }    
+        throw new UnsupportedOperationException("ReplicatorQueue.createContentFilterChain not supported and is deprecated.");
+    }
 
     /**
      * @return the deactivateOperations

